cylc-snippets.json

```diff
@@ -0,0 +1,672 @@
+    },
+    "[directives]": {
+        "prefix": "[directives]",
+        "body": "[[[directives]]]",
+        "description": "Job runner (batch scheduler) directives.\n\nThe default value is set in the global config: :cylc:conf:`global.cylc[platforms][<platform name>][directives]`.\n\nSupported for use with job runners:\n\n- pbs\n- slurm\n- loadleveler\n- lsf\n- sge\n- slurm_packjob\n- moab\n\nDirectives are written to the top of the job script in the correct format\nfor the job runner.\n\nSpecifying directives individually like this allows use of default directives\nfor task families which can be individually overridden at lower levels of the\nruntime namespace hierarchy."
+    },
+    "<directive>": {
+        "prefix": "<directive>",
+        "body": "<directive> = ${1:*value unset*}",
+        "description": "Example directives for the built-in job runner handlers are shown in\n:ref:`AvailableMethods`."
+    },
```

Comment:
Ah, should probably strip the "many" settings by detecting items that begin and end with `<`/`>`